### PR TITLE
FChecker Cresencio

### PR DESCRIPTION
**Pull request by carlacresencio**
IT COMPILESSSS

### CODE REVIEW

**Review by jjesuscastro — COMMENTED** [review]
does this work?

File: include/FChecker.h [hunk 1]
```diff
@@ -4,4 +4,6 @@ class FChecker : public LetterChecker
 {	
 	//override the function "checkMyLetter"	
 	//create the function declaration here
+	public: 	
+		char checkMyLetter(char inputChar);
 };
```
Comment (jjesuscastro) [inline include/FChecker.h]: DO THIS:


#include "LetterChecker.h"

class FChecker : public LetterChecker
{	
	//override the function "checkMyLetter"	
	//create the function declaration here	
	char checkMyLetter(char inputChar) override;
};

File: src/FChecker.cpp [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
 //override the function "checkMyLetter"
+char FChecker::checkMyLetter(char inputChar) 
+{
+	char letter; 
+	letter = 'f';
+	printf("haha fuk");
+	return letter;
+}
+
 //create the function definition here
```
Comment (jjesuscastro) [inline src/FChecker.cpp]: THEN DO THIS

#include "FChecker.h"

//always returns defaultLetter :P	
char FChecker::checkMyLetter(char inputChar)
{
	//std::cout << "\nDefault letter '*' used." << std::endl;
	if(inputChar == 'F' || inputChar == 'f')
	{		
		return 'f';
	}
	else
		return defaultLetter;
}

File: include/FChecker.h (hunk 1 above)
Comment (carlacresencio) [reply include/FChecker.h]: i copy paste that in FChecker.h?

File: src/FChecker.cpp (hunk 2 above)
Comment (carlacresencio) [reply src/FChecker.cpp]: WHAT A BLESSING HHAHHAHAAH

**Comment by jjesuscastro** [comment]
Yay

**Review by jjesuscastro — APPROVED** [review]
It works!

**Pull request merged**